03/app.test.js

```diff
@@ -1 +1,38 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+describe('RandomNumber', () => {
+    it('return 1 if min and max = 1', () => {
+        expect(randomNumber(1, 1)).toBe(1);
+    });
+
+    it('throw exception when min is not a number', () => {
+        expect(() => {
+            randomNumber('some string', 2);
+        }).toThrow();
```

Comment:
Tutaj należy napisac jakiego komunikatu się spodziewamy, aby być pewnym, że jest to o co nam chodzil.
Sam błąd to za mało bo przecież mogą być również.

PS. Oczywiście obecnie ta treść będzie się mocno powielać, ale trzeba to zmienić.